src/services/tracing.ts

```diff
@@ -30,7 +29,7 @@ propagation.setGlobalPropagator(new W3CTraceContextPropagator())
 
 // FYI this hook is executed BEFORE the `formatError` hook from apollo
 // The data.errors field here may still change before being returned to the client
-const gqlResponseHook = (span: Span, data: graphqlTypes.ExecutionResult) => {
+const gqlResponseHook = (span: Span, data) => {
```

Comment:
@opentelemetry/instrumentation-graphql use a different graphql version to validate types